processor/geoipprocessor/testdata/attribute_source_address/input-logs.yaml

```diff
@@ -0,0 +1,23 @@
+resourceLogs:
+  - resource:
+      attributes:
+        - key: ip
+          value:
+            stringValue: 1.2.3.4
```

Comment:
Could we make this IP different than the one in the log so we know which IP is used as the input?

Reply:
Definitely, if I understood correctly, this would suggest changing the attributes with an IP that won't be used as input to ease troubleshooting/testing. Changed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34214/commits/5ad475b8d8775ddafd5728b30990cf794074e274